templates/services/docker-compose_base.tmpl

```diff
@@ -33,6 +33,7 @@ services:
         max-file: "10"{{end}}{{end}}
 {{template "consensus" .}}
 {{template "validator" .}}
+{{template "validator-blocker" .}}
```

Comment:
This executes before the validator service, therefore it might be better to put it before the validator service and after the consensus one, as the script right now is organized in the order of the compose execution